### DISCUSSION TITLE
ValueError: Provided model path does not exist. Please check the path or provide a model_url to download.

### DISCUSSION THREAD

**Discussion by dinubogdan1973** [imartinez/privateGPT#1572] [Q&A]
Hello everyone,

I'm trying to install privateGPT and i'm stuck on the last command : 

`poetry run python -m private_gpt`

I got the message "ValueError: Provided model path does not exist. Please check the path or provide a model_url to download."

Few details:

Python 3.11.5
pyenv 3.1.1
Poetry (version 1.7.1)

Someone can help me please ?

PS: I found that I have 2 folders in pypoetry/Cache/virtualenvs/: 
my-project-0302-xxxxxxxx-py3.11 (named when poetry installed)
private-gpt-xxxxxxxx-py3.11

**Comment by justinomora** [comment]
Hello @dinubogdan1973! I believe I encountered this problem recently. Please run: `poetry install --with local` and then once that's done, run: `poetry run python -m private_gpt`. Please let me know if that works!

**Reply by Obfusgated** [reply]
not working for me
